### ISSUE TITLE
Trim is not working well

### ISSUE THREAD

**Issue by realtica** [jqlang/jq#3078]
Please test it in: [jqplay](https://jqplay.org/jq?q=trim%2C%20ltrim%2C%20rtrim&j=%22%20abc%20%22)

_The error is general:_
jq: error: trim/0 is not defined at <top-level>, line 1:
trim, ltrim, rtrim
jq: error: ltrim/0 is not defined at <top-level>, line 1:
trim, ltrim, rtrim      
jq: error: rtrim/0 is not defined at <top-level>, line 1:
trim, ltrim, rtrim             
jq: 3 compile errors
exit status 3

**Comment by wader** [comment]
Hey! jqplay uses jq 1.7.1 and the trim functions are current only in the development version of jq and will most likely be in next release.

**Comment by wader** [comment]
It would be nice if jqplay could support different versions of jq, maybe even a "nightly" somehow. But that discussion should probably be at https://github.com/owenthereal/jqplay

**Comment by wader** [comment]
Maybe the main confusion comes from that https://jqlang.github.io/jq/manual/ is docs for the development version and not latest stable release?

**Comment by rrrnld** [comment]
Oh, I agree. This bit me today as well! I used the manual as documentation and tried it in the command line, which didn't work. I assumed that the manual is for the latest stable version and that I had some older version installed. I tried it in the playground -- same result, which confused me even more. My next thought was that maybe I'm looking docs for a fork, or that the docs were generated by an LLM and unreliable (oh no)… only to find this issue. Maybe it makes sense to keep the docs for the stable version, and offer a nightly version of the docs when explicitly asked?
